package com.hspedu.exception_;

public class Exception01 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
int num1 = 10;
int num2 = 0;
try {
	int res = num1 / num2;
} catch (Exception e) {
	// TODO: handle exception
	//e.printStackTrace();
}
System.out.println("程序继续进行。。");

	
	}

}
